Makefile

```diff
@@ -10,7 +10,7 @@ PREFIX ?= ${DESTDIR}/usr/local
 BINDIR ?= ${PREFIX}/bin
 LIBEXECDIR ?= ${PREFIX}/libexec
 MANDIR ?= ${PREFIX}/share/man
-ETCDIR ?= /etc
+ETCDIR ?= ${DESTDIR}/etc
```

Comment:
why wouldnt this be PREFIX?

Reply:
No Prefix is usually DESTDIR/usr or DESTDIR/usr/local